aws/rust-runtime/aws-config/test-data/default-credential-provider-chain/imds_assume_role/http-traffic.json

```diff
@@ -89,7 +89,7 @@
             "action": {
                 "Request": {
                     "request": {
-                        "uri": "http://169.254.169.254/latest/meta-data/iam/security-credentials/",
+                        "uri": "http://169.254.169.254/latest/meta-data/iam/security-credentials-extended/",
```

Comment:
So this test is updated to hit extended but the response isn't updated?

Reply:
Ah, good point, not strictly necessary since extended API can still return credentials without account ID. Admittedly, this is what happens when we manually tweak connection recording files for a feature that hasn't been supported in the real world.